specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,214 @@
+{  
+   "swagger":"2.0",
+   "info":{  
+      "version":"1",
+      "title":"SubscriptionCreationClient",
+      "description":"Subscription creation client provides an interface to create, modify and retrieve azure subscriptions programmatically."
+   },
+   "schemes":[  
+      "https"
+   ],
+   "host":"management.azure.com",
+   "consumes":[  
+      "application/json"
+   ],
+   "produces":[  
+      "application/json"
+   ],
+   "paths":{  
```

Comment:
Every RP needs a top-level "operations" operation:

"/providers/Microsoft.Subscription/operations": {
      "get": {
        "tags": [
          "Subscriptions"
        ],
        "description": "Lists all of the available Microsoft.Subscription API operations.",
        "operationId": "Operations_List",
        "parameters": [
          {
            "$ref": "#/parameters/apiVersionParameter"
          }
        ],
        "responses": {
          "200": {
            "description": "OK. The request has succeeded.",
            "schema": {
              "$ref": "#/definitions/OperationListResult"
            }
          },
          "default": {
            "description": "Error response describing why the operation failed.",
            "schema": {
              "$ref": "#/definitions/ErrorResponse"
            }
          }
        },
        "x-ms-pageable": {
          "nextLinkName": "nextLink"
        }
      }
    },

Reply:
You can look at other swaggers and pretty much copy/paste this from there.